test/BotTester.spec.ts

```diff
@@ -321,6 +321,20 @@ describe('BotTester', () => {
     });
 //```
 
+//```javascript
+    it('can filter messages out', () => {
```

Comment:
copy this section of code to the README.md. Theres no auto-gen script at the moment